property_app/views.py

```diff
@@ -0,0 +1,69 @@
+from django.shortcuts import render
+from django.contrib.auth import authenticate
+from .models import Property, Portfolio
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework.status import (
+    HTTP_201_CREATED,
+    HTTP_404_NOT_FOUND,
+    HTTP_204_NO_CONTENT,
+    HTTP_401_UNAUTHORIZED
+)
+from rest_framework.authtoken.models import Token
+from rest_framework.authentication import TokenAuthentication
+from rest_framework.permissions import IsAuthenticated
+from django.contrib.auth.hashers import check_password
+
+# Create your views here.
+
+class Get_Portfolio(APIView):
+
+    authentication_classes = [TokenAuthentication]
+    permission_classes = [IsAuthenticated]   
+
+    def get(self, request):
+        user = request.user
+        portfolio = Portfolio.objects.get(user=user)
+
+        return Response(True)
+
+class A_Property(APIView):
+
+    authentication_classes = [TokenAuthentication]
+    permission_classes = [IsAuthenticated]   
+
+    def post(self, reqeuest):
+        pass
+
+    def get(self, request, id):
+        pass
+
+    def delete(self, request, id):
```

Comment:
This is looking great, I do see a bit of repeated code in your views. Remember at the root level this is still OOP so you could utilize Inheritance to pass your permission classes to all of your views. Maybe in utilities.py you create a class named ViewWithPermissions 
`
class ViewWithPermissions(APIView):
  authentication_classes = [TokenAuthentication]
  permission_classes = [IsAuthenticated]
` 
and then just use inheritance everywhere else
`
class All_Lists(ViewWithPermissions):
  def get(self, request):
    pass
`

`

Reply:
This didn't format the way I wanted it too lmk if you'd like some further clarification